### ISSUE TITLE
Removing this cookbook?

### ISSUE THREAD

**Issue by hCraker** [ProjectPythiaCookbooks/mpl-colorbar-cookbook#13]
In [this PR](https://github.com/ProjectPythia/pythia-foundations/pull/310) and it's associated issue, it was determined that the information in this cookbook was better suited for the pythia-foundations page. Now that the info has been moved over, do we still need this cookbook?

@jukent @brian-rose

**Comment by jukent** [comment]
No need for duplicate information in both places! But if you have other ideas of how to expand this Cookbook, it could still be valuable.

**Comment by hCraker** [comment]
I'm leaning towards deleting it since I don't have any plans to add to it. Would removing the cookbook be as simple as deleting/archiving this repository, or do I need to change some files elsewhere?

**Comment by jukent** [comment]
Go ahead and archive it. On the main repo, it will need to be removed from the gallery yaml file.

**Comment by brian-rose** [comment]
Yes, I suggest archiving but no need to delete! You never know when someone might want to pick it up again.

**Comment by hCraker** [comment]
I don't think I have the privileges to archive this (I can't see the settings tab). Could one of you archive it for me?
